react-app/src/api/distributionAPI.ts

```diff
@@ -81,6 +86,33 @@ export const useDistributionAPI = (): IDistributionAPI => {
     };
   }, [wallet, query]);
 
+  const getDelegationRewardsByValidators = useCallback(
+    async (delegatorAddress: string, validatorAddresses: string[]) => {
+      const rewardPromises: Promise<QueryDelegationRewardsResponse>[] = [];
+
+      validatorAddresses.forEach((validatorAddress) =>
+        rewardPromises.push(
+          query.distribution.delegationRewards(
+            delegatorAddress,
+            validatorAddress
+          )
+        )
+      );
+      const rewards = await Promise.all(rewardPromises);
+      return rewards.map((rewardRespond) => {
+        const reward = rewardRespond.rewards.find(
+          (r) => r.denom === CoinMinimalDenom
+        );
+        const rewardAmount = new BigNumber(reward?.amount ?? 0).shiftedBy(-18);
```

Comment:
Maybe leave a comment here so when people who are unfamiliar with this would know why we need to shiftBy -18